subprojects/common/elastic/src/main/kotlin/com/avito/android/elastic/HttpElasticClient.kt

```diff
@@ -60,9 +61,13 @@ internal class HttpElasticClient(
 
             params.putAll(metadata)
 
+            val formattedDate = isoDate.format(now)
+
+            dateFormatChecker.check(formattedDate)
+
             elasticApi.log(
                 indexPattern = indexPattern,
-                date = isoDate.format(nowWithTimeZone),
+                date = formattedDate,
```

Comment:
sad that elastic can't accept a date in Long format

Reply:
it is index in url path